src/css/profile/mobile/common/slider.less

```diff
@@ -82,21 +62,18 @@
 	}
 
 	&.ui-disabled {
+		opacity: 1;
```

Comment:
Is it necessary to set the opacity with 1?

Reply:
No needed. I removed. 
Thanks